src/components/Header/Header.jsx

```diff
@@ -1,3 +1,89 @@
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import SHeader from './Style';
+
 export default function Header() {
-  return <div>Header</div>;
+  const [translations, setTranslations] = useState({});
+  const [images, setImages] = useState({});
+
+  useEffect(() => {
+    axios.get('http://localhost:8080/texts').then(({ data }) => {
```

Comment:
You shouldn't use a hardcoded "http://localhost:8080" value: once in production, it won't work anymore.
Create a `.env` in your frontend folder and move this value there, same as the backend!